dev/AutoSuggestBox/AutoSuggestBox_themeresources.xaml

```diff
@@ -315,6 +315,8 @@
                             IsHorizontalRailEnabled="{TemplateBinding ScrollViewer.IsHorizontalRailEnabled}"
                             IsVerticalRailEnabled="{TemplateBinding ScrollViewer.IsVerticalRailEnabled}"
                             IsDeferredScrollingEnabled="{TemplateBinding ScrollViewer.IsDeferredScrollingEnabled}"
+                            VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+                            HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
```

Comment:
I think it makes sense to add those bindings to the regular textbox style too.

Reply:
By regular - you mean? If you mean the actual textbox template code, it's still in the OS :)

Reply:
The default TextBox template has been pulled out of the OS into WinUI: https://github.com/microsoft/microsoft-ui-xaml/blob/626c9be99f6cccf8c59b19cdfa5bd245563762fa/dev/CommonStyles/TextBox_themeresources.xaml#L91

Reply:
@Felix-Dev beat me to it. Yes the default TextBox template (and RichEditBox template) have been lift out of the OS into WInUI 2.

Reply:
@Felix-Dev thank you for pointing this out :) Didn't know about it! I'll have a look in a few:)